docs/openapi/api-external-b2b-pa-bundle.yaml

```diff
@@ -1642,285 +1662,341 @@ components:
           pattern: ^[0-9]{6}$
           type: string
+          description: end date of the period
       description: Reporting Period (Sezione INPS) object
     InpsRecord:
       allOf:
         - $ref: '#/components/schemas/IncludeNotificationCost'
       type: object
       properties:
-        officeCode:
+        office:
           pattern: ^[0-9]{3,4}$
           type: string
-        contributionReason:
+          description: identification code of the office
+        reason:
           pattern: ^[A-Z-]{3,4}$
           type: string
-        inpsCode:
+          description: contribution reason for the record
+        inps:
           pattern: ^[A-Za-z0-9\s]{0,17}$
           type: string
+          description: INPS identification code
         period:
           $ref: '#/components/schemas/Period'
-        debitAmount:
+        debit:
           pattern: ^[0-9]{3,15}$
           type: string
-        creditAmount:
+          description: debit amount of the record
+        credit:
           pattern: ^[0-9]{3,15}$
           type: string
+          description: credit amount of the record
       description: INPS Record object
     InpsSection:
       type: object
       properties:
-        inpsRecordList:
+        records:
           type: array
           items:
             $ref: '#/components/schemas/InpsRecord'
+          description: INPS Record List
       description: INPS Section (Sezione INPS) object
     RegionRecord:
       allOf:
         - $ref: '#/components/schemas/IncludeNotificationCost'
       type: object
       properties:
-        regionCode:
+        region:
           pattern: ^[0-9]{2}$
           type: string
-        taxTypeCode:
+          description: region identification code
+        taxType:
           pattern: ^[0-9A-Z]{4}$
           type: string
+          description: identification code of the type of tax
         installment:
           pattern: ^[A-Z0-9]{0,4}$
           type: string
+          description: identification code of the ente
         year:
           pattern: ^[1-2][0-9]{3}$
           type: string
-        debitAmount:
+          description: reference year
+        debit:
           pattern: ^[0-9]{3,15}$
           type: string
-        creditAmount:
+          description: debit amount of the record
+        credit:
           pattern: ^[0-9]{3,15}$
           type: string
+          description: credit amount of the record
       description: Region Record object
     RegionSection:
       type: object
       properties:
-        regionRecordList:
+        records:
           type: array
           items:
             $ref: '#/components/schemas/RegionRecord'
+          description: Region Record List
       description: Region Section (Sezione Regioni) object
     LocalTaxRecord:
       allOf:
         - $ref: '#/components/schemas/IncludeNotificationCost'
       type: object
-      required:
-        - institutionCode
-        - section
-        - taxTypeCode
```

Comment:
Perche sono stati eliminati?

Reply:
@MTurraPpa il file che ha aggiornato Chiappini in data 21 Luglio ha riportato questa modifica